frontend/components/RepoCardItem/style.css

```diff
@@ -0,0 +1,3 @@
+.repo__card:hover {
+  box-shadow: 10px 10px 5px 0px rgba(0, 0, 0, 1);
+}
```

Comment:
- I forgot to descript the hove effect. I updated it. Check this [link](https://github.com/m3ntorship/m3ntorship.com/issues/207)